README.md

```diff
@@ -389,11 +450,11 @@ Sample response:
 }
 ```
 
-### Invoke-connection
+### InvokeConnection
 
 Using  InvokeConnection, end-user applications can integrate checkout/card issuance flow with their apps/systems. To invoke connection, use the invokeConnection(config ConnectionConfig) method of the Skyflow client. The config object must have `connectionURL`,`methodName` and remaining are optional. 
```

Comment:
Change:

> Using InvokeConnection, end-user applications can integrate checkout/card issuance flow with their apps/systems.

End-user apps can use InvokeConnection to integrate checkout and card issuance flows with their apps and systems.

> To invoke connection

To invoke a connection

> and remaining are optional.

and the remaining are optional.

Reply:
Done